test/rlcIntegration/securityAadRest.spec.ts

```diff
@@ -0,0 +1,9 @@
+import SecurityAADRestClient from "./generated/securityAADRest/src";
+import { assert } from "chai";
+
+describe("Security AAD Rest Client", () => {
+  it("should be able to instantiate a new client without Typescript errors", () => {
+    const client = SecurityAADRestClient("subscriptionId" as any);
```

Comment:
I wasn't very sure how to create a fake TokenCredential here.

Reply:
I think you should be able to pass an object like

{getToken: async () => {assert.equal(validate, something); return { accessToken: "FakeToken"}}}

Reply:
Updated.